src/librustc_mir/transform/qualify_consts.rs

```diff
@@ -355,10 +365,13 @@ impl<'a, 'tcx> Qualifier<'a, 'tcx, 'tcx> {
                 TerminatorKind::FalseUnwind { .. } => None,
 
                 TerminatorKind::Return => {
+                    if self.tcx.sess.features_untracked().const_let {
+                        break;
```

Comment:
I would instead put an `if` around all the code below, just to make sure it's explicit.